pypxe/dhcp.py

```diff
@@ -274,6 +271,9 @@ def craft_options(self, opt53, client_mac):
             dns_server = b''.join([socket.inet_aton(i) for i in dns_server])
             response += self.tlv_encode(6, dns_server)
             response += self.tlv_encode(51, struct.pack('!I', 86400)) # lease time
+            if self.broadcast:
+                broadcast = self.get_namespaced_static('dhcp.binding.{0}.broadcast'.format(self.get_mac(client_mac)), self.broadcast)
+                response += self.tlv_encode(28, socket.inet_aton(broadcast))
```

Comment:
Setting the [broadcast TLV](https://datatracker.ietf.org/doc/html/rfc2132#section-5.3) shouldn't be necessary. Hosts can figure it out for themselves.